packages/web-frontend/cypress/config/dashboardReportUrls/default.json

```diff
@@ -324,39 +324,39 @@
     "dashboard": "WASH"
   },
   {
-    "code": "LESMIS_repetition_rate_primary_district",
+    "code": "LESMIS_age_of_grade1_entrance",
```

Comment:
Apart from fixing wrong dashboard names, I sorted some items that were out of order.

The implied order in config files is by ascending `code` - this generally makes it easier to compare with other configs, find what is/is not included, which org unit variations are used for the same code etc